src/modules/feedback/components/toggleFeedback/ToggleFeedbackPanel.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @module modules/feedback/components/toggleFeedback/ToggleFeedbackPanel
+ */
+import { html } from 'lit-html';
+import { $injector } from '../../../../injection';
+import { MvuElement } from '../../../MvuElement';
+import css from './toggleFeedbackPanel.css';
+import { classMap } from 'lit-html/directives/class-map.js';
+
+/**
+ * possible feedback types
+ * @enum
+ */
+export const FeedbackType = Object.freeze({
+	NONE: 'none',
+	MAP: 'map',
+	GENERAL: 'general'
+});
+
+const Select_Feedback_Type = 'select_feedback_type';
+
+export class ToggleFeedbackPanel extends MvuElement {
+	constructor() {
+		super({
+			selectedFeedbackPanel: FeedbackType.NONE
+		});
+
+		const { TranslationService: translationService } = $injector.inject('TranslationService');
+
+		this._translationService = translationService;
+	}
+
+	update(type, data, model) {
+		switch (type) {
+			case Select_Feedback_Type:
+				return { ...model, selectedFeedbackPanel: data };
+		}
+	}
+
+	createView(model) {
+		const { selectedFeedbackPanel } = model;
+		const translate = (key) => this._translationService.translate(key);
+
+		const feedbackTypeMap = () => {
+			this.signal(Select_Feedback_Type, FeedbackType.MAP);
+		};
+
+		const feedbackTypeGeneral = () => {
+			this.signal(Select_Feedback_Type, FeedbackType.GENERAL);
+		};
```

Comment:
The wrapper-functions for the signal-calls cannot hide or simplify complexity nor gives a better naming for the signal-calls. So i prever to get rid of them.

Instead increase the readability by using of `...@click=${() => this.signal(Select_Feedback_Type, FeedbackType.MAP)}`